src/issue.ts

```diff
@@ -8,6 +8,25 @@ interface Label {
 }
 
 export class Issue extends TreeItem {
+    contextValue = 'issue';
+    original_issue? : Issue;
```

Comment:
Why do we need a `original issue` here?

Reply:
Just want to keep the original object "in case of", not really used or useful indeed.